packages/framework/.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: php
 
 php:
-    - 7.1
```

Comment:
i think ` - echo '' | pecl upgrade redis` was included just because of php7.1 travis image

Reply:
https://travis-ci.org/shopsys/framework/builds/540619776/config

Reply:
Even for 7.2 and 7.3 it is updated. I'm not sure if it can be safely dropped.

Reply:
yes, pleas drop